junit2htmlreport/runner.py

```diff
@@ -3,6 +3,7 @@
 """
 import os
 from argparse import ArgumentParser
+from multipledispatch import dispatch
```

Comment:
this isn't present in setup.py

Reply:
yes , i am currently implementing a coroutine on my end and calling junit2html asynchronously. But junit2html seems designed in such a way to make it run from cmdline only and generally i feel is not good practice to modify sys.argv values on the fly. This patch adds some  flexibility so that an user can call it asynchronously without having to create a blocking subprocess (or non-blocking). By overloading the start() method , this gives user the option whether he wants to run junit2html from cmdline or as a coroutine function. 
for multipledispatch , can you like add a requirements for the project or add in the readme for user to locally install or something . i am not sure how to handle that . thanks again for reviewing

Reply:
why don't you just import the `run()` function and call that with args?

Reply:
Can you give an example of how you'd invoke this? is it from other python code?

Reply:
Google for `from junit2htmlreport` there are quite a few examples, eg https://twitter.com/pythononwheels/status/1091086690175893510?s=20